### PR TITLE
Feat/cz/accordion

### PR DESCRIPTION
**Pull request by CarlZac**
## Summary
- Accordion layout.
- Button component applied.

## Screenshot
image

## Trello Card
https://trello.com/c/CIqUdl3Q/26-layout-accordion-component

### CODE REVIEW

File: src/app/components/accordion/accordion.component.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import '/src/scss/colors';
+
+.container {
+  margin-top: 20px;
+  width: 100%;
+  .accordion {
```
Comment (miguelhenaoa) [inline src/app/components/accordion/accordion.component.scss:6]: Add white line

File: src/app/components/accordion/accordion.component.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      cursor: pointer;
+      display: flex;
+      font-size: 14px;
+      font-weight: 400;
+      justify-content: space-between;
+      .icon {
+        margin-left: 80px;
+        width: 15px;
+        height: 15px;
+        transition: transform 0.2s ease-in-out;
```
Comment (miguelhenaoa) [inline src/app/components/accordion/accordion.component.scss:21]: Sort

File: src/app/components/accordion/accordion.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <div class="accordion" *ngFor="let panel of panels; let i = index">
+    <div class="heading" (click)="togglePanel(i)">
+      <span>{{ panel.title }}</span>
+      <img class="icon" [src]="upArrow" [class.rotate]="panel.toggle" />
```
Comment (miguelhenaoa) [inline src/app/components/accordion/accordion.component.html:5]: We should use `<i>` instead `` tag, and add class with background image

File: src/app/components/accordion/accordion.component.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
+@import '/src/scss/colors';
+
+.container {
+  margin-top: 20px;
+  width: 100%;
+  .accordion {
+    border-bottom: 1px solid $concrete;
+    margin: 10px 60px;
+    padding-bottom: 10px;
+    .heading {
```
Comment (miguelhenaoa) [inline src/app/components/accordion/accordion.component.scss:10]: Should be `header`

**Review by miguelhenaoa — CHANGES_REQUESTED** [review]

File: src/app/components/accordion/accordion.component.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+@Component({
+  selector: 'app-accordion',
+  templateUrl: './accordion.component.html',
+  styleUrls: ['./accordion.component.scss']
+})
+
+export class AccordionComponent{
+  @Input() panels: AccordionItem[] = [];
+  upArrow = 'assets/images/png/up-arrow.png';
```
Comment (andres-acc) [inline src/app/components/accordion/accordion.component.ts:12]: This is not necessary anymore.

**Review by andres-acc — APPROVED** [review]

**Pull request merged**